open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-spring/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/server/spring/AnalyticsModelingOMASResource.java

```diff
@@ -148,4 +150,24 @@ public AnalyticsModelingOMASAPIResponse createArtifact(
 
 		return restAPI.createArtifact(serverName, userId, serverCapability, artifact);
 	}
+
+    /**
+	 * Update assets in repository defined by artifact (body of the request).
+     * @param serverName  unique identifier for requested server.
+     * @param userId      request user
+	 * @param serverCapability where the artifact is stored.
+	 * @param artifact from json definition.
+	 * @return errors or list of created assets.
+	 */
+    @Operation(summary = "Update assets representing analytics artifact.")
+	@PutMapping(path = "/sync")
+	public AnalyticsModelingOMASAPIResponse updateArtifact(
+			@PathVariable("serverName") String serverName,
+            @PathVariable("userId") String userId,
+			@RequestParam(required=true) String serverCapability,
```

Comment:
What is serverCapability? the GUID? The qualifiedName?

Reply:
the serverCapability is string uniquely identifying the storage of analytic artifact. It could be URL where the storage can be accessed.